packages/sampler/stories/qa/VirtualListNative.js

```diff
@@ -1,29 +1,56 @@
 import React from 'react';
 import Item from '@enact/moonstone/Item';
-import {VirtualListNative} from '@enact/moonstone/VirtualList';
+import {VirtualListNative, VirtualListBase} from '@enact/moonstone/VirtualList';
 import {Column, Cell} from '@enact/ui/Layout';
+import ri from '@enact/ui/resolution';
+import {ScrollableBaseNative as UiScrollableBaseNative} from '@enact/ui/Scrollable/ScrollableNative';
+import {VirtualListBaseNative as UiVirtualListBaseNative} from '@enact/ui/VirtualList/VirtualListBase';
 
+import {action} from '@storybook/addon-actions';
 import {storiesOf} from '@storybook/react';
 
+import {boolean, number, select} from '../../src/enact-knobs';
+import {mergeComponentMetadata} from '../../src/utils/propTables';
+
+const Config = mergeComponentMetadata('VirtualListNative', UiVirtualListBaseNative, UiScrollableBaseNative, VirtualListBase);
+
+const
+	defaultDataSize = 10,
```

Comment:
Do we need to define `defaultDataSize` because the `defaultDataSize` is used once?

Reply:
No, I don't. I'll fix it.